ui/app/models/job.js

```diff
@@ -327,4 +328,16 @@ export default class Job extends Model {
   // spec first. In order to preserve both the original HCL and the parsed response
   // that will be submitted to the create job endpoint, another prop is necessary.
   @attr('string') _newDefinitionJSON;
+
+  @computed('variables', 'parent', 'plainId')
+  get pathLinkedVariable() {
```

Comment:
`praise`:  Great job handling the behavior of parameterized and periodic child jobs.